File: languages/go/8kyu/if-you-cant-sleep-just-count-sheep.go
// Title: If you can't sleep, just count sheep!!
// Link: https://www.codewars.com/kata/5b077ebdaf15be5c7f000077
// Difficulty: 8 kyu
//
// ## Description
//
// Given a non-negative integer n, return a string "1 sheep...2 sheep...n sheep...".
// For n == 0, return an empty string.

package kata

import (
	"strconv"
	"strings"
)

func countSheep(num int) string {
	var b strings.Builder
	for i := 1; i <= num; i++ {
		b.WriteString(strconv.Itoa(i))
		b.WriteString(" sheep...")
	}
	return b.String()
}

File: languages/python/8kyu/if-you-cant-sleep-just-count-sheep.py
"""
Title: If you can't sleep, just count sheeps!!
Link: https://www.codewars.com/kata/5b077ebdaf15be5c7f000077
Difficulty: 8 kyu

## Description

If you can't sleep, just count sheeps!!

**Task**:
Given a non-negative integer, `3` for example, return a string with a murmur: `"1 sheep...2 sheep...3 sheep..."`. Input will always be valid, i.e. no negative integers.
"""

def count_sheep(n):
    return ''.join(f"{i} sheep..." for i in range(1, n + 1))